### PR TITLE
fix: input validation alert for start & end date

### PR DESCRIPTION
**Pull request by amabelleS**
# Description
Showing an alert when the end date is later than the start date - on the dashboard and gaps-pattern pages.

## screenshots
image

image

related issue: #309 (not closing)

### CODE REVIEW

**Comment by amabelleS** [comment]
@ArkadiK94 How can I solve this? 

image

File: yarn.lock
Comment (NoamGaash) [inline yarn.lock:1]: this file should not be added - we no longer work with yarn

File: src/locale/he.json [hunk 1]
```diff
@@ -97,6 +97,7 @@
   "bug_type_message": "אנא הזן סוג בקשה!",
   "bug_type_bug": "באג",
   "bug_type_feature": "בקשה לפיתוח",
+  "bug_date_alert": "תאריך הסיום אינו יכול להיות מאוחר יותר מתאריך ההתחלה",
```
Comment (NoamGaash) [inline src/locale/he.json:100]:
```suggestion
  "bug_date_alert": "תאריך הסיום אינו יכול להיות מוקדם יותר מתאריך ההתחלה",
```

File: yarn.lock
Comment (ArkadiK94) [reply yarn.lock:1]: We should change the error message because it is misleading

Comment (amabelleS) [reply yarn.lock:1]: > We should change the error message because it is misleading

What should we write in the error message? Why is it misleading?

Comment (amabelleS) [reply yarn.lock:1]: > this file should not be added - we no longer work with yarn

if I installed all the dependencies with yarn on this project, can I just switch to npm from now?

Comment (ArkadiK94) [reply yarn.lock:1]: I meant the error message in the github actions,  from "after removing package-lock.json file" to "after removing yarn.lock file"
However, it is not related to this pr...

**Review by NoamGaash — APPROVED** [review]

**Comment by NoamGaash** [comment]
great job! :muscle:

**Review by shootermv — APPROVED** [review]

**Pull request merged**